routes/web.php

```diff
@@ -1,18 +1,16 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
+use \App\Http\Controllers\Admin\LoginController;
+use \App\Http\Controllers\Admin\HomeController;
+use \App\Http\Controllers\Admin\RegisterController;
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
+Route::get('login', [LoginController::class,'showLoginForm'])->name('login');
```

Comment:
```suggestion
Route::get('login', [LoginController::class, 'showLoginForm'])->name('login');
```
Check convention của các Route khác nữa